app-crypt/keybase/keybase-1.0.48.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils systemd user versionator
```

Comment:
where are you using `versionator` rather than `eapi7-ver` and where are you using `eutils`?

Reply:
dunno, please advise how to fix it

Reply:
`MY_PV=$(replace_version_separator 3 '-')`